common/containers/ParitySignerDecrypt/index.scss

```diff
@@ -0,0 +1,14 @@
+.ParitySignerUnlock {
+    a img {
+        display: inline-block;
+        height: 3em;
+        margin: 0 0.4em;
+    }
+
+    .qr-bounds {
```

Comment:
We use [SuitCSS naming convention](https://github.com/MyCryptoHQ/MyCrypto/wiki/Style-Guides#classnames) to avoid class name conflicts. Likewise, the `a img` above would be better suited by class names.